python/pyspark/ml/tests.py

```diff
@@ -238,6 +238,17 @@ def setParams(self, seed=None):
         return self._set(**kwargs)
 
 
+class HasThrowableProperty(Params):
```

Comment:
I realized I kind of placed this in a weird place in the file previously, so I moved it next to the test case that was using it to be consistent with the other tests.  It's not related to this PR, so would you prefer I move it back?